packages/site/src/layout/SearchBox.tsx

```diff
@@ -126,6 +129,39 @@ export const SearchBox = ({
                 </ContentCardWrapper>
               </Content>
             )}
+            {filteredComponentList.length == 0 &&
+              filteredDesignList.length == 0 && (
```

Comment:
There is probably a way to extract this out into a variable or function or whatever you'd call it, happy to do that if someone can give me a hand.